draft-ietf-rats-eat.md

```diff
@@ -1102,94 +1102,46 @@ The values for the results enumerated type are as follows:
 
 ### Submodules (submods) {#submods}
 
-Some devices are complex, having many subsystems.  A
-mobile phone is a good example. It may have several connectivity
-subsystems for communications (e.g., Wi-Fi and cellular). It may have
-subsystems for low-power audio and video playback. It may have multiple
-security-oriented subsystems like a TEE and a Secure Element.
+Some devices are complex and have many subsystems.  A mobile phone is a good example. It may have subsystems for communications (e.g., Wi-Fi and cellular), low-power audio and video playback, multiple
```

Comment:
I think it would be good for the new text you are bringing in for submods.

Probably half the document is one line per sentence now.  RATS Architecture is about 50/50 too.

Reply:
I will adopt this practice for new text as edits are made.